### PR TITLE
Remove unused mate_panel_applet_position_menu ()

### PR DESCRIPTION
**Pull request by wmww**
As far as I can tell, this was zombie code.

### CODE REVIEW

File: mate-panel/applet.h [hunk 1]
```diff
@@ -111,11 +111,6 @@ void        mate_panel_applet_menu_set_recurse (GtkMenu     *menu,
 
 gboolean    mate_panel_applet_toggle_locked  (AppletInfo *info);
 
-void mate_panel_applet_position_menu (GtkMenu   *menu,
```
Comment (muktupavels) [inline mate-panel/applet.h]: Is not this API break? Should not *.so version of library be updated to reflect this?

Comment (wmww) [reply mate-panel/applet.h]: What library is this part of? I thought it was just part of the `mate-panel` application.

Comment (muktupavels) [reply mate-panel/applet.h]: Oh, sorry... I was responding based on function name / prefix that is used by `libmate-panel-applet`.

Comment (wmww) [reply mate-panel/applet.h]: Ah, yes, that is confusing. Clearing that up is another benefit of this PR.

**Comment by raveit65** [comment]
Indeed, looks like that mate_panel_applet_position_menu is a left over from
https://github.com/mate-desktop/mate-panel/commit/002244141bd0205aff0aa3e7028529d1ba2ab22c
and
https://github.com/mate-desktop/mate-panel/commit/8001d65ccfbb4fccd84407680e4e4da5dc16cde4
Thanks, i forgot to remove it.

For external applets the code is here https://github.com/mate-desktop/mate-panel/blob/master/libmate-panel-applet/mate-panel-applet.c#L839
IHMO.

**Pull request merged**